src/js/rest/postDeploy.js

```diff
@@ -0,0 +1,32 @@
+import createError from './common/createError';
+import errors from './resource/errorMessages.json';
+import sendRequest from './common/sendRequest';
+
+/** Function: postDeploy
+ *  @param {object} params
+ *  @param {object[]} params.apps
+ *  @param {number} params.apps[].app
+ *  @param {number?} params.revision
```

Comment:
Because revision comes with a particular app, so I think this line of code should be changed to something like params.apps[].revision